tests/kafkatest/tests/streams/streams_broker_compatibility_test.py

```diff
@@ -29,7 +29,7 @@ class StreamsBrokerCompatibility(Test):
     These tests validates that
     - Streams works for older brokers 0.11 (or newer)
     - Streams w/ EOS-alpha works for older brokers 0.11 (or newer)
-    - (TODO) Streams w/ EOS-beta works for older brokers 2.5 (or newer)
+    - Streams w/ EOS-beta works for older brokers 2.5 (or newer)
```

Comment:
Maybe I misunderstood this TODO here, but I suppose EOS-beta should only work with broker 2.5 or higher correct? This sentence is a bit weird @mjsax

Reply:
Well, I hear you, but it's correct: We test Streams `2.6` against "old" broker `2.5` -- also note, that we will extend this test in the future and test Streams `2.7` against `2.5` and `2.6` brokers and so forth, ie, we test the _current_ version against all _older_ brokers that support eos-beta.

Does this make sense?

Reply:
Oh I see that now.